src/App.js

```diff
@@ -1,16 +1,29 @@
-import { BrowserRouter } from "react-router-dom";
+import { BrowserRouter, Switch, Route } from "react-router-dom";
```

Comment:
Normally we rename the `BrowserRouter` as `Router`.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#import_an_export_with_a_more_convenient_alias

Reply:
I changed to BrowserRouter as Router.